index.d.ts

```diff
@@ -255,7 +255,7 @@ declare class Conf<T = any> implements Iterable<[keyof T, T[keyof T]]> {
 
 	@param object - A hashmap of items to set at once.
 	*/
-	set(object: Partial<T>): void;
+	set(object: T): void;
```

Comment:
This has to be `Partial`, right? Since it should be possible to pass in only a subset of the types properties. Can you add a type test with a strongly-typed store for it?

Reply:
Yes, this should accept a subset, not the whole `T`

Reply:
Done!